generic/samples/native-client/build.gradle

```diff
@@ -0,0 +1,33 @@
+plugins {
+    id "org.jetbrains.kotlin.multiplatform"
+}
+
+repositories {
+    mavenCentral()
+}
+
+kotlin {
+    ext.detectTarget = { os ->
+        if (os == "Mac OS X") return macosX64('native')
```

Comment:
switch case?

Reply:
How to make a switch for `else if (os.startsWith("Windows")) return mingwX64("native")`?

Reply:
You could take the first part 
```
switch(os.split(' ')[0]) {
case "Mac":
case "Linux":
case "Windows":
default: 
}
```

but not sure that it's necessary